src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1259,7 +1259,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
         return;
     }
 
-    bool isTrusted = app_.getUNL ().nodeInUNL (signerPublic);
+    auto const isTrusted = app_.validators().trusted (signerPublic);
```

Comment:
I didn't realize it until I saw this changed line, but the new function name `trusted` really clarifies intent.